fw-child/resources/scss/_template.scss

```diff
@@ -56,6 +51,15 @@
   pointer-events: none;
 }
 
+.header-fixed-menu {
+  &.on-white {
+    #header-menu-icon {
+      color: $black;
+      transition: color 500ms;
```

Comment:
Apply this transition property on #header-menu-icon before the .on-white class is added for fluidity